installation/sd-cards.md

```diff
@@ -4,7 +4,7 @@ The Raspberry Pi should work with any compatible SD card, although there are som
 
 ## SD card size (capacity) 
 
-For installation of NOOBS or the image installation of Raspbian, the minimum recommended card size is 8GB. For Raspbian Lite image installations we recommend a minimum of 4GB. Some distributions, specifically LibreELEC and Arch, can run on much smaller cards. If you're planning to use a card of 64GB or more with NOOBS, see [this page](sdxc_formatting.md) first.
+For installation of Raspbian with desktop and recommended software (Full) via NOOBS the minimum card size is 16GB. For the image installation of Raspbian with desktop and recommended software, the minimum card size is 8GB. For Raspbian Lite image installations we recommend a minimum of 4GB. Some distributions, for example LibreELEC and Arch, can run on much smaller cards. If you're planning to use a card of 64GB or more with NOOBS, see [this page](sdxc_formatting.md) first.
```

Comment:
To be pedantic, I guess this _could_ say "For the image (or NOOBS Lite) installation of Raspbian with desktop and recommended software, the minimum card size is 8GB." but that's possibly too much complication? :man_shrugging: